jobs/JGDAS_FIT2OBS

```diff
@@ -0,0 +1,83 @@
+#! /usr/bin/env bash
+
+source "${HOMEgfs}/ush/preamble.sh"
+source "${HOMEgfs}/ush/jjob_header.sh" -e "fit2obs" -c "base fit2obs"
+
+
+##############################################
+# Set variables used in the script
+##############################################
+
+export CDATE=${CDATE:-${PDY}${cyc}}
+export CDUMP=${CDUMP:-${RUN:-"gdas"}}
+
+export xdate=$(${NDATE} -${VBACKUP_FITS} ${CDATE})
+export CDATE=${xdate} # Reset CDATE to xdate for lookback
+export vday=$(echo ${xdate} | cut -c1-8)
+export vcyc=$(echo ${xdate} | cut -c9-10)
+
+export COM_INA=${ROTDIR}/gdas.${vday}/${vcyc}/atmos
+export COM_INF='$ROTDIR/vrfyarch/gfs.$fdy/$fzz'
```

Comment:
I notice this is in `'` quotes, so `$fdy` and `$fzz` won't get resolved here.
Where are these used?

Reply:
The `COM_INF` needs to be passed into the Fit2Obs package with single quotes and then be evaluated there when `fdy` and `fzz` are set. I tried setting this with double quotes (based on shellcheck warning) but then it tries to evaluate it at the workflow level which fails since `fdy` and `fzz` aren't known yet.

Reply:
Made a suggestion to suppress the shellcheck warning

Reply:
Works for me, thanks @WalterKolczynski-NOAA !